kujira/assetlist.json

```diff
@@ -1155,6 +1155,1134 @@
+            },
+            {
+                "denom": "lp dydx-mnta",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira13g2eelkf9q4md4xx52ylqvlgpccn50f2vyu05eazufd2g4jatmhq2425pq/ulp",
+        "name": "LP Bow: DYDX-MNTA",
+        "display": "lp dydx-mnta",
+        "symbol": "LP DYDX-MNTA",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+            }
+        ]
+    },
+    {
+        "description": "LP Bow: wAVAX.axl-MNTA",
+        "denom_units": [
+            {
+                "denom": "factory/kujira15pk8c09vcsgsdq4xlh2qfqxtu2r8pgdy7nggkqj8p3h2phke80pqwnvtzk/ulp",
+                "exponent": 0
+            },
+            {
+                "denom": "lp wavax-mnta",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira15pk8c09vcsgsdq4xlh2qfqxtu2r8pgdy7nggkqj8p3h2phke80pqwnvtzk/ulp",
+        "name": "LP Bow: wAVAX.axl-MNTA",
+        "display": "lp wavax-mnta",
+        "symbol": "LP wAVAX.axl-MNTA",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+                "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+            }
+        ]
+    },
+    {
+        "description": "LP Bow: CHEQ-MNTA",
+        "denom_units": [
+            {
+                "denom": "factory/kujira15w6jhxsv924mn9cfr0qg9z36hltmw4rzd68x40d9re6hzyhslumqjw9zqd/ulp",
+                "exponent": 0
+            },
+            {
+                "denom": "lp cheq-mnta",
+                "exponent": 6
+            }
+        ],
+        "base": "factory/kujira15w6jhxsv924mn9cfr0qg9z36hltmw4rzd68x40d9re6hzyhslumqjw9zqd/ulp",
+        "name": "LP Bow: CHEQ-MNTA",
+        "display": "lp cheq-mnta",
+        "symbol": "LP CHEQ-MNTA",
+        "logo_URIs": {
+            "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
+            "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.svg"
+        },
+        "images": [
+            {
+                "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kujira/images/bow.png",
```

Comment:
Due to the different backgrounds (navy vs transparent), bow.svg and bow.png are not visually identical and must be registered under separate image objects. So only one of these can be primary and may go into logo_URIs